charter.md

```diff
@@ -2,92 +2,82 @@
 
 ## Introduction
 
-Digital credentials are essential to identity, authorization, licenses, certificates and digitization use cases that are part of modernization efforts targeting efficiency, transparency, such as digital licenses or certificates of origin.
-IETF WGs and IRTF RGs have provided COSE and JOSE based building blocks.  Things like identifiers, claim structures, and confidentiality for personally identified information enable interoperability and cryptographic agility, however, they have required profiling in order to render digitization easy, unambiguous and consistent.
-The many ways to combine identifiers, key material, signatures, and attributes to create credentials, often prevents interoperabtility from spanning across data silos or cross use cases.
-This lack of interoperabilty leads to increased implementation costs, attack surface, and harms adoption, which prevents digitization from being deployed successfully.
-SPICE aligns the capabilities of JOSE and COSE to support modern cryptographic capabilities, such as selective disclosure or unlinkablilty, consistent identity attribute semantics across representations, and Internet-scale discoverability of identifiers and cryptography capabilities.
+Digital credentials may be thought as digital equivalents of paper-based passports, driver's licenses, membership certificates or proofs of qualification attached to a person.
+These types of credentials usually contain personal data such as a family name, first names, a gender, a birth date, an home address, a phone number, a face picture
+or minutiae points for fingerprints.
 
-## Background
-
-IETF and IRTF working groups have developed foundational building blocks with BBS Signatures, RSA Blind Signatures, Verifiable Random Functions, or other Selective-Disclosure and data minimization techniques.
-While these techniques are being incorporated into JOSE and OAuth-based protocols, the integration into protocols using CBOR/COSE is, however, missing. 
-The SPICE WG aims to close this gap by enabling the secure and privacy-friendly use of CBOR-based credentials for use cases that go beyond constrained Internet of Things devices. Envisioned use cases where these credentials will be used include education, digital wallets, business-to-business supply chain interactions, and digital media.
-
-### A note on terminology (Note: context to be removed from charter upon adoption)
+In some contexts, persons may be willing either to partially disclose some values of their attributes or to demonstrate some properties about their attributes without disclosing their values.
+When received by an entity, a proof of ownership of the digital credential needs to be provided and verified, so that only the legitimate owner of the digital credential can take an advantage of its possession. The use of Trusted Applications (TAs) supported by a Trusted Execution Environment (TEE) is essential in order to provide a proof of ownership of a digital credential.
```

Comment:
```suggestion
When received by an entity, a proof of the digital credential needs to be provided and verified, so that only the legitimate controller of the digital credential can take an advantage of its possession. The use of Trusted Applications (TAs) supported by a Trusted Execution Environment (TEE) is essential in order to provide a proof of control of a digital credential.
```

Reply:
Is "use of Trusted Applications (TAs) supported by a Trusted Execution Environment (TEE) is essential in order to provide a proof of control of a digital credential" essential at this juncture and explicit in the charter? I know of one or two parties are assertive on this. I do not think this is a hard requirement for the charter. Should proponents work in this WG, TEEP, and other relevant groups for *later* documents? Sure. Does this mean all architectures and related deliverables must mandate TEE out the gate? It is not 100% so in this section, but we are getting close enough. Is that the most important thing now?

Reply:
I don't think we need this language - feels overcomplicated for the charter and too specific

Reply:
The use of a TA (under a TEE) is necessary in order to prevent a collusion / collaborative attack between Alice and Bob.

As an example if Alice (12) is colluding with Bob (25), whatever cryptography is being used, it is impossible to prevent Alice to get some data from Bob demonstrating that she is over 18.

Preventing the export of the value of a private key or a secret key, does not prevent its use, at least, by the legitimate owner of these keys. At the time a credential request is initiated, the holder must demonstrate either that a private key or a link secret is stored in a TEE and originates from a given TA. 

Then after, these keys can only be used by that TA that does not allow any operation to be done with these keys, but only specific operations on already formatted data.

Without TEEs, collusion / collaborative attackz cannot be prevented.

Reply:
I might be okay with "may be necessary", but I don't think this is a needed restriction.